tests/sles4sap/trento/deploy_trento.pm

```diff
@@ -25,8 +28,10 @@ sub run {
 
     my $script_run = ' ./';
     # Run the Trento deployment
+
+    record_info('00.040');
```

Comment:
What does this `00.040` mean ?

Reply:
It is the ID of the deployment script, it is part of the name of the deployment script

Reply:
Thank you